connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/ConnectAssertions.java

```diff
@@ -524,7 +524,7 @@ public void assertConnectorIsStopped(String connectorName, String detailMessage)
      * @param connectorState
      * @param numTasks the expected number of tasks
      * @param tasksState
-     * @return true if the connector and tasks are in RUNNING state; false otherwise
+     * @return true if the connector and tasks are in expected states; false otherwise
```

Comment:
```suggestion
     * @return true if the connector and tasks are in the expected state; false otherwise
```
nit: they're all expected to be in the same state